examples/three-validators/docker-compose.yml

```diff
@@ -53,7 +53,8 @@ services:
     environment:
       VALIDATOR_DATABASE_HOST: alice_db
     volumes:
-      - ./alice/config/application.yml:/src/hedera-eth-bridge-validator/application.yml
+      - ./alice/config/bridge.yml:/src/hedera-eth-bridge-validator/config/bridge.yml
```

Comment:
Can't we have only one `bridge.yml` config for the 3 node network and pass it to all 3 validators?